scripts/tsconfig.json

```diff
@@ -6,8 +6,7 @@
     "noEmitOnError": true,
     "noImplicitAny": true,
     "strictNullChecks": true,
-    "typeRoots": ["node_modules/@types"],  // "typeRoots": [] -> won't work
-    "types": ["node"]                      // "types": [] -> won't work
+    "skipLibCheck": true
```

Comment:
i'm not usually one to skip things but we do the same skip in the other repo's tsconfig https://github.com/5e-bits/5e-srd-api/blob/a726d196023803ef0a533640d8b0f4103cff96d1/tsconfig.json#L91